web_source/references.bib

```diff
@@ -186,6 +186,14 @@ @article{etoori2021a
   pages        = {e29},
   date         = 2022,
 }
+@article{feng2024a,
+  title        = {Data Resource Profile: A national linked mother-baby cohort of health, education and social care data in England (ECHILD-MB)},
+  author       = {Feng, Q. and Ireland, G. and Gilbert, R. and Harron, K.},
+  journal      = {Int J Epidemiol},
+  volume       = {53},
+  pages        = {3},
+  date         = {2024},
+}
```

Comment:
Maybe also add doi and url elements.

Reply:
yes, will do, I was focusing on getting the basics right! I will do as you suggest. I will also add the second Feng article. thank you for your help!

Reply:
Absolutely, understand. It's looking great. Thanks for getting involved 👍